rllib/utils/replay_buffers/episode_replay_buffer.py

```diff
@@ -631,16 +630,6 @@ def _update_add_metrics(
                 reduce="mean",
                 window=self._metrics_num_episodes_for_smoothing,
             )
-            # Number of module steps in the buffer.
-            module_steps_stored = self.metrics.peek(
-                (NUM_MODULE_STEPS_ADDED_LIFETIME, mid)
-            ) - self.metrics.peek((NUM_MODULE_EPISODES_EVICTED_LIFETIME, mid))
-            self.metrics.log_value(
-                (NUM_MODULE_STEPS_STORED, mid),
-                module_steps_stored,
-                reduce="mean",
-                window=self._metrics_num_episodes_for_smoothing,
-            )
```

Comment:
This metric is confusing because we don't checkpoint replay buffers by default.
Every time you train from a checkpoint and train for a while, this metric will go up by an amount equal to the capacity of the buffer.